add RSE job advert to jobs page (#91)
OxfordRSE/website: content/page/jobs.md

---
title: "Jobs"
date: 2023-13-06T12:13:10Z
draft: false
---

The OxRSE group currently has multiple open positions for
Research Software Engineers (RSE). This is an exciting opportunity to join a
growing team and work as an RSE on a diverse selection of world-leading and
impactful projects. We offer RSEs the potential to advance their career and to
make a significant contribution to the development of good software practices at
a world-leading university. 

These posts are offered as open-ended positions. They are full time roles, but

> flexible options will be considered (e.g. part-time, job share). We are also

Author: ```suggestion
flexible options will be considered (e.g. part-time, job share).
```

---
title: "Jobs"
date: 2023-13-06T12:13:10Z
draft: false
---

The OxRSE group currently has multiple open positions for
Research Software Engineers (RSE). This is an exciting opportunity to join a
growing team and work as an RSE on a diverse selection of world-leading and
impactful projects. We offer RSEs the potential to advance their career and to
make a significant contribution to the development of good software practices at
a world-leading university. 

> These posts are offered as an open-ended positions. They are full time roles, but

Author: ```suggestion
These posts are offered as open-ended positions. They are full time roles, but
```

flexible options will be considered (e.g. part-time, job share). We are also

> open to remote work options within the UK.

Author: ```suggestion
```

You will work on a variety of world-class research projects across the University of
Oxford, deliver workshops and training in software development best practices,
and further develop your own technical and research skills as well as those
outside of software development, such as project management and consultancy.

We are looking for candidates with significant experience in software
development within a research environment (a doctoral degree is preferred, but

> equivalent experience will also be considered) and who have have a demonstrated

Author: ```suggestion
equivalent experience will also be considered) and who have a demonstrated
```

ability to design, implement and deploy a complex software engineering project,

> individually or as part of a team. You must be able to collaborate effectively

Author: ```suggestion
both individually and as part of a team. You must be able to collaborate effectively
```

with other researchers and have a deep understanding of software engineering
best practices (design, testing, documentation, reproducibility etc.) and how to
apply these effectively and to champion their use within a research environment.

For more details please see the [Job Description](0018702339.pdf). Applications

> can be made be emailing [Martin Robinson](mailto:martin.robinson@dtc.ox.ac.uk)

Author: ```suggestion
can be made by emailing [Martin Robinson](mailto:martin.robinson@dtc.ox.ac.uk)
```
